content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -37,6 +37,12 @@ The upgrade workflow at high level is the following:
 - Make sure you read the [release notes]({{< latest-release-notes >}}) carefully.
 - Make sure to back up any important components, such as app-level state stored in a database.
   `kubeadm upgrade` does not touch your workloads, only components internal to Kubernetes, but backups are always a best practice.
+- If you're running CentOS, RHEL, Fedora or directly use the
+  [yum.k8s.io](https://yum.k8s.io) repositories: The `kubernetes-cni` package
+  was obsoleted with the release of Kubernetes v1.18.4, v1.17.7 and v1.16.11 on
```

Comment:
nit: Should this type of information appear here or in the release notes?

Reply:
See https://github.com/kubernetes/website/pull/21898/files#r443414063